locked-asset/token-unstake/src/fees_accumulation.rs

```diff
@@ -72,7 +71,8 @@ pub trait FeesAccumulationModule:
         );
 
         let payment = self.call_value().single_esdt();
-        let locked_token_id = self.get_locked_token_id(&energy_factory_addr);
+        let locked_token_id =
```

Comment:
To better isolate this somewhat low-level code, `get_locked_token_id` should internally use `LOCKED_TOKEN_ID_STORAGE_KEY`, not have it as argument.

Reply:
Done.